helper-cli/src/main/kotlin/commands/repoconfig/GenerateScopeExcludesCommand.kt

```diff
@@ -193,6 +193,11 @@ private fun getScopeExcludesForPackageManager(packageManagerName: String): List<
                 reason = ScopeExcludeReason.TEST_DEPENDENCY_OF,
                 comment = "Packages for code analysis (testing) only."
             ),
+            ScopeExclude(
+                pattern = "spotbugs",
+                reason = ScopeExcludeReason.BUILD_DEPENDENCY_OF,
```

Comment:
Rereading this, I'm wondering why we use `BUILD_DEPENDENCY_OF` here, but for other linters like PMD we use `TEST_DEPENDENCY_OF`. Actually, I believe both are not good fits for linters, and all of them should use `DEV_DEPENDENCY_OF` instead. What do you think @fviernau?

Reply:
For `findbugs` we also us `BUILD_DEPENDENCY_OF`. Anyhow, fully agree on your proposal.
I don't mind if this commit remains as is, IMO it can be fixed in a separate commit for all linters.

Reply:
Ideally, this PR would get a new first commit fixing it for all linters, and then *spotbugs* gets added with the correct reason right away. Just to ensure we'll not forget about improving this.

Reply:
@bennati would you implement that?

Reply:
ok, can you provide a list of linters?

Reply:
All of the following are lintes / static code analysis tools and should use `ScopeExcludeReason.DEV_DEPENDENCY_OF` (some of them already do): "checkstyle", "detekt", "findbugs", "ktlint", "lint.*", "pmd", "spotbugs"

Reply:
done